docs/cli/dotnet.md

```diff
@@ -0,0 +1,78 @@
+dotnet command
+==============
+
+## NAME
+
+dotnet -- general driver for running the command-line commands
+
+## SYNOPSIS
+
+dotnet [--version] [--help] [--verbose] < command > [< args >]
+
+## DESCRIPTION
+dotnet is a generic driver for the CLI toolchain. Invoked on its own, it will give out brief usage instructions. 
+
+Each specific feature is implemented as a command. In order to use the feature, it is specified after dotnet, i.e. `dotnet compile`. All of the arguments following the command are command's own arguments.  
+
+
+## OPTIONS
+`-v, --verbose`
+
+Enable verbose output.
+
+`--version`
+
+Print out the version of the CLI tooling
+
+`-h, --help`
+
+Print out a short help and a list of current commands. 
+
+## DOTNET COMMANDS
+
+The following commands exist for dotnet.
+
+* [dotnet-new](dotnet-new.md)
+   * Initializes a sample .NET Core console application. 
+* [dotnet-restore](dotnet-restore.md)
+  * Restores the dependencies for a given application. 
+* [dotnet-build](dotnet-build.md)
+  * Compile the application to either an intermidiate language (IL) or to a native binary. 
+* [dotnet-run](dotnet-run.md)
+   * Runs the application from source.
+* [dotnet-publish](dotnet-publish.md)
+   * Publishes a flat directory that contains the application and its dependencies, including the runtime binaries. 
+* [dotnet-test](dotnet-test.md)
+   * Runs tests using a test runner specified in project.json.
+* [dotnet-pack](dotnet-pack.md)
+   * Create a NuGet package of your code
+
```

Comment:
Yeah, we need to fix that. No REPL in the package. Wait, I have a PR outstanding with that.